pkg/receive/handler.go

```diff
@@ -61,6 +70,9 @@ func (h *Handler) Receive(w http.ResponseWriter, r *http.Request) {
 
 	defer r.Body.Close()
 
+	// Limit the request body size to a sane default
+	r.Body = http.MaxBytesReader(w, r.Body, requestLimit)
```

Comment:
This request limit should really be configurable via a command line argument so that we don't need to PR a golang change to bump the limit.

Reply:
We should also be able to disable the limi, eg if the limit is 0

Reply:
This limiter should also be its own Middleware i think. It doesn't really need to be couped to the logic of forwarding requests.

Reply:
This limiter should also be its own Middleware i think. It doesn't really need to be couped to the logic of forwarding requests.